sqlest/src/main/scala/sqlest/ast/syntax/UntypedReads.scala

```diff
@@ -0,0 +1,72 @@
+package sqlest.ast.syntax
+
+import org.joda.time.{ DateTime, LocalDate }
+import scala.util.Try
+import sqlest.ast._
+import sqlest.util.Iso8601
+
+trait UntypedReads[A] {
```

Comment:
You're absolutely right - this is left over from a different implementation really